src/OpenTelemetry/Metrics/MetricAggregators/ISumMetric.cs

```diff
@@ -0,0 +1,31 @@
+// <copyright file="ISumMetric.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Collections.Generic;
+
+namespace OpenTelemetry.Metrics
+{
+    internal interface ISumMetric : IMetric
+    {
+        bool IsDeltaTemporality { get; }
+
+        bool IsMonotonic { get; }
+
+        IEnumerable<IExemplar> Exemplars { get; }
+
+        object Sum { get; }
```

Comment:
Wouldn't Sum metrics benefit from `IDataPoint`?

Reply:
IDataPoint has Tags.

However, we can refactor out the Value portion from IDataPoint if we think that would be useful. Something like a IDataValue.

Reply:
Unless this is blocking, let's tackle this with an Issue/PR after this as necessary.